### ISSUE TITLE
about iff-net

### ISSUE THREAD

**Issue by sister-tong** [YUCHEN005/DPSL-ASR#6]
Hi author, I noticed that you only have the program to train dpsl in your project, what should I do if I just want to train iff-net without dpsl?

**Comment by YUCHEN005** [comment]
Hi, thanks for interest. Please see https://github.com/YUCHEN005/DPSL-ASR/blob/master/espnet2/layers/iff_net.py for network code of iffnet.

For training with iffnet-only, you may modify the training code (forward-prop, loss functions) to keep iffnet branch only: https://github.com/YUCHEN005/DPSL-ASR/blob/master/espnet2/asr/dpsl_asr.py#L149-L342. 

Since this work is completed long time ago, the original code cannot be retrieved now, sorry for inconvenience.
